spec/site/tlc/alarm_spec.rb

```diff
@@ -74,6 +74,40 @@
         end
       end
     end
+    
+    it 'can be suspended', :script, sxl: '>=1.0.7' do |example|
+      
+      alarm_code_id = 'A0301'   # what alarm to expect
+      component = 'KK+AG9998=001DL003'
+      alarm_status = /inActive/i
+      timeout  = Validator.config['timeouts']['alarm']
+
+      Validator::Site.connected do |task,supervisor,site|
+        log "Check that alarm suspend #{alarm_code_id} can be send and confirmed"
+        m_id = RSMP::Message.make_m_id  # generate a message id, that can be used to listen for repsonses
+        alarm = RSMP::AlarmSuspend.new(
```

Comment:
is is valid to suspend an alarm that is not active? i guess it might be, but worth checking.
a potential problem here is that if the alarm is already suspended, you will not get an alarm when suspending it. to make it more robust, i suggest doing an initial resume before moving on the the actual test.

Reply:
another potential issue is that you expect the alarm status to be /inActive/i, but that will fail if the alarm is active when the test is running.